PERFORMANCE.md

```diff
@@ -17,7 +17,7 @@ bin/apicast
 For profiling with stapxx it is recommended to start just one process and worker:
 
 ```shell
-bin/apicast -m off -w 1 -c examples/configuration/echo.json  > /dev/null
+bin/apicast -m off -w 1 -c $(pwd)/examples/configuration/echo.json  > /dev/null
```

Comment:
This change is not required after https://github.com/3scale/apicast/pull/471 is merged.

Reply:
It's already merged, so you can change this.